tests/manage/test_create_pod_attaching_multiple_PVC_start_IO.py

```diff
@@ -0,0 +1,180 @@
+"""
+Test Case OCS-260, Create Cirros POD with multiple PVC attached and Start IO
+"""
+
+import logging
+import pytest
+import random
+import time
+
+from ocs_ci.ocs import constants, exceptions
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs.resources import pvc
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from tests import helpers
+from ocs_ci.framework import config
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def ocs260_fixture(request):
+
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def setup():
+    """
+    Setting up the environment for the test
+    """
+    log.info("Create Ceph Block Pool")
+    global RBD_POOL
+    RBD_POOL = helpers.create_ceph_block_pool(pool_name="cirrostest")
+
+    log.info("Create RBD Secret")
+    global RBD_POOL_SECRET_OBJ
+    RBD_POOL_SECRET_OBJ = helpers.create_secret(constants.CEPHBLOCKPOOL)
+
+    log.info("Create RBD Storage Class")
+    global RBD_STORAGE_CLASS_OBJ
+    RBD_STORAGE_CLASS_OBJ = helpers.create_storage_class(
+        constants.CEPHBLOCKPOOL,
+        interface_name=RBD_POOL.name,
+        secret_name=RBD_POOL_SECRET_OBJ.name
+    )
+
+
+def teardown(self):
+    """
+    Tearing down the environment
+    """
+    dc = ocp.OCP(
+        kind=constants.DEPLOYMENTCONFIG, namespace=config.ENV_DATA['cluster_namespace']
+    )
+    dc.delete(resource_name=self.cirros_name[0], wait=True)
+    pvc.delete_all_pvcs()
+    RBD_STORAGE_CLASS_OBJ.delete()
+    RBD_POOL_SECRET_OBJ.delete()
+    RBD_POOL.delete()
+
+
+def create_pvc(pvc_name_prefix, pvc_count):
+    """
+    Function to create multiple PVC based on count and name
+
+    Args:
+        pvc_name_prefix (str): User defined pvc prefix name
+        pvc_count (int): Specify number of pvc count to be create
+
+    Returns:
+        pvc_list (list): Returns the pvc created
+    """
+    pvc_list = []
+    for count in range(pvc_count):
+        name = pvc_name_prefix + str(count)
+        pvc_obj = helpers.create_pvc(sc_name=RBD_STORAGE_CLASS_OBJ.name, pvc_name=name)
+        pvc_list.append(pvc_obj.name)
+    return pvc_list
+
+
+def create_pod(pvc_list):
```

Comment:
I think you can break down the implementation here to pod creation (which you can take from ocs_ci.ocs.resources.pod.Pod) and as Raz suggested below, attach pvc to pod one